.docker/images/govcms8/settings/production.settings.php

```diff
@@ -39,3 +39,6 @@
     $config['environment_indicator.indicator']['name'] = 'Edit Domain';
   }
 }
+
+// Disable temporary file deletion (GOVCMSD8-576).
+$config['system.file']['temporary_maximum_age'] = 0;
```

Comment:
Can the value be defined in an environment variable?

Reply:
At this time no, we should raise a separate ticket if that is important.
See #135 for full details.